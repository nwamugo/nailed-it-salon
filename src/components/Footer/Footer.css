.Footer {
  position: absolute;
  width: 100%;
  height: 130px;
  left: 0px;
  top: 2308px;
  background: black;
}
.foot {
  font-style: normal;
  font-weight: 500;
  font-size: 18px;
  line-height: 21px;
  text-align: center;
  color: white;
}
i {
  font-size: 35px;
  margin-top: 20px;
}
