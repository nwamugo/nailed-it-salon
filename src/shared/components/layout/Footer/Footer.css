.Footer {
  position: absolute;
  width: 100%;
  height: 8.125rem;
  top: 144.25rem;
  background: black;
}

.foot {
  font-style: normal;
  font-weight: 500;
  font-size: 1.125rem;
  line-height: 1.3125rem;
  text-align: center;
  color: white;
}

.row {
  width: 100%
}

i {
font-size: 2.1875rem;
margin-top: 1.25rem;
  color: white;
}

a.social {
  color: black;
}
