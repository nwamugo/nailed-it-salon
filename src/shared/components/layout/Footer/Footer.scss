.Footer {
  position: absolute;
  width: 100%;
  height: 13rem;
  top: 230rem;
  background: black;
}

.foot {
  font-style: normal;
  font-weight: 500;
  font-size: 1.8rem;
  line-height: 2.1rem;
  text-align: center;
  color: white;
}

.Footer .row {
  width: 100%
}

i {
  font-size: 3.5rem;
  margin-top: 2rem;
  color: white;
}

a.social {
  color: black;
}
