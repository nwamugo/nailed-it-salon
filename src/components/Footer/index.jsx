/* eslint-disable no-irregular-whitespace */
import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <div className="Footer">
      <div className="row">
        <div className="foot col-md-12">
          <i className="fab fa-facebook" />
          <span>&nbsp;&nbsp;&nbsp;</span>
          <i className="fab fa-instagram" />
          <span>&nbsp;&nbsp;&nbsp;</span>
          <i className="fab fa-twitter" />
          <br />
          <br />
          <span>Copyright &copy; 2019Â by Nail Salon</span>
        </div>
      </div>
    </div>
  );
};

export default Footer;
