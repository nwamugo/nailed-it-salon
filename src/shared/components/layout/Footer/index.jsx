/* eslint-disable no-irregular-whitespace */
import React from 'react';
import './Footer.scss';
import NailLogo from '../../../images/nailLogo.png';

const Footer = () => {
  return (
    <div className="Footer">
      <div className="row">
        <div className="foot col-md-12">
          <img src={NailLogo} width="130px" className="foot-logo" alt="logo" />
          <br />
          <a
            href="https://www.snapchat.com/add/n_naileditsa"
            className="social"
            rel="noopener noreferrer"
          >
            <i className="fab fa-snapchat" />
          </a>
          <span>&nbsp;&nbsp;&nbsp;</span>
          <a
            href="https://www.instagram.com/20naileditsa/"
            className="social"
            rel="noopener noreferrer"
          >
            <i className="fab fa-instagram" />
          </a>
          <span>&nbsp;&nbsp;&nbsp;</span>
          <br />
          <br />
          <span className="foot-copyright">Copyright &copy; 2020</span>
        </div>
      </div>
    </div>
  );
};

export default Footer;
