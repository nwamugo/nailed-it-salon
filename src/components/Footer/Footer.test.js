import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index';

describe('Footer', () => {
  it('should render correctly', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
