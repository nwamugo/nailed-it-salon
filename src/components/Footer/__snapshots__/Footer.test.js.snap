// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div
  className="Footer"
>
  <div
    className="row"
  >
    <div
      className="foot col-md-12"
    >
      <i
        className="fab fa-facebook"
      />
      <span>
           
      </span>
      <i
        className="fab fa-instagram"
      />
      <span>
           
      </span>
      <i
        className="fab fa-twitter"
      />
      <br />
      <br />
      <span>
        Copyright © 2019 by Nail Salon
      </span>
    </div>
  </div>
</div>
`;
